build.gradle

```diff
@@ -314,99 +314,116 @@ if (BuildUtils.shouldPublish(project) || BuildUtils.shouldPublishDistribution(pr
 }
 
 subprojects {
-    task("showRepos", group: "Help", description: "Show the list of repositories currently in use.").doLast({
-        repositories.each {
-            println "repository: ${it.name} (${it.hasProperty("url") ? it.url : it.getDirs()})"
-        }
-    })
-    project.task("showConfigs", group: "help", description: "Show all the configurations available in the project and their published artifacts")
-        .doLast({
-            project.configurations.forEach({
-                Configuration config ->
-                    println(config.name + ": (${config.getDescription()})")
+    project.tasks.register("showRepos", DefaultTask) {
+        task ->
+            task.group = "Help"
+            task.description = "Show the list of repositories currently in use."
+            task..doLast({
```

Comment:
Is this `..` some special syntax or just a typo?

Reply:
Typo. Thanks.